packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -106,9 +106,9 @@ export default class Meeting extends StatelessSparkPlugin {
   setMeetingInfo(meetingInfo) {
     // MeetingInfo will be undefined for 1:1 calls
     if (meetingInfo) {
-      this.convoId = this.meetingInfo.convoId;
-      this.locusUrl = this.meetingInfo.locusUrl;
-      this.sipUri = this.meetingInfo.sipMeetingUri;
+      this.convoId = this.meetingInfo.convoId || this.convoId;
```

Comment:
When would we have `this.convoId` but `meetingInfo.convoId` would be undefined?

Reply:
1:1 calls